static/dev/animations.html

```diff
@@ -1289,7 +1289,7 @@
 				response.toVerse = cv[1]
 			} else if (cv.length == 1) {
 				response.toVerse = cv[0]
-				response.toChaptere = response.chapter
+				response.toChapter = response.chapter
```

Comment:
This could be a bug fix (or as noted elsewhere introduce a bug if something wasn't expecting this code to function)